src/main/java/nl/tudelft/contextproject/util/FileUtil.java

```diff
@@ -82,13 +82,13 @@ public static File getFile(String location) {
 			
 			return new File(url.toURI());
 		}
-		
+
 		//Check if we already have the file extracted
 		String winLoc = fLocation.replace('/', File.separatorChar);
 		File file = new File(winLoc);
 		if (file.exists()) return file;
 
-		//We need to extract this from the jar.
+		//If does file has not yet been extracted we extract it
```

Comment:
Old comment was proper english, this one is not xD.

Did you mean "File has not yet been extracted, so we extract it."